.ci/docker/common/install_xpu.sh

```diff
@@ -1,26 +1,30 @@
 #!/bin/bash
 set -xe
 
```

Comment:
hi @chuanqi129 could you please review these changes in this file.

Reply:
Thanks @atalman LGTM